src/test/java/com/fasterxml/jackson/core/write/UTF8GeneratorTest.java

```diff
@@ -105,6 +106,36 @@ void surrogatesWithRaw() throws Exception
         jp.close();
     }
 
+    @Test
```

Comment:
Will move to test class that verifies #223 handling (where `COMBINE_UNICODE_SURROGATES_IN_UTF8` implemented).